.github/workflows/test.yaml

```diff
@@ -150,10 +143,22 @@ jobs:
       with:
         go-version-file: go.mod
 
+    - name: Determine GoReleaser ID
+      id: goreleaser_id
+      shell: bash
+      run: |
+        if [ "${{ matrix.operating-system }}" == "windows-latest" ]; then
+          echo "id=build-windows" >> $GITHUB_OUTPUT
+        elif [ "${{ matrix.operating-system }}" == "macos-latest" ]; then
+          echo "id=build-macos" >> $GITHUB_OUTPUT
+        else
+          echo "id=build" >> $GITHUB_OUTPUT
+        fi
+
     - name: Run GoReleaser
       uses: goreleaser/goreleaser-action@v4
       with:
         version: v1.16.2
-        args: release --skip-sign --snapshot --clean --skip-publish --timeout 90m
+        args: build --snapshot --clean --timeout 90m --id ${{ steps.goreleaser_id.outputs.id }}
```

Comment:
I'm not sure about change `release` -> `build` subcommand, but removing `--skip-sign` looks dangerous, isn't it?